src/actions/search.js

```diff
@@ -40,3 +40,8 @@ export const removeDateRange = () => ({
 export const resetDateRange = () => ({
   type: types.RESET_DATE_RANGE,
 });
+
+export const changeSortMethod = sortMethodValue => ({
+  type: types.CHANGE_SORT_METHOD,
+  payload: { sortMethodValue },
```

Comment:
If there is only one parameter, maybe you can just pass like this:
```
payload: sortMethodValue
```

Reply:
I just a little bit confused about the ... syntax, so this won't be changed in today's PR due to time limitaion